### PR TITLE
Convert common.sage to Python

### PR DESCRIPTION
**Pull request by divergentdave**
This converts `common.sage` to Python, as a first incremental step towards #204. This required two `^` changes, and then adjustments to import statements in other files.

### CODE REVIEW

File: poc/prg.sage [hunk 1]
```diff
@@ -5,10 +5,10 @@ from Cryptodome.Cipher import AES
...
 from Cryptodome.Util import Counter
 from Cryptodome.Util.number import bytes_to_long
-from sagelib.common import TEST_VECTOR, VERSION, Bytes, Error, Unsigned, \
-                           format_custom, zeros, from_le_bytes, gen_rand, \
-                           next_power_of_2, print_wrapped_line, to_be_bytes, \
-                           to_le_bytes, xor, concat
+from common import TEST_VECTOR, VERSION, Bytes, Error, Unsigned, \
+                   format_custom, zeros, from_le_bytes, gen_rand, \
+                   next_power_of_2, print_wrapped_line, to_be_bytes, \
+                   to_le_bytes, xor, concat
```
Comment (cjpatton) [inline poc/prg.sage:11]: FWIW I started moving each imported thing onto its own light, in order to more easily maintain alphabetical order:
```python
from common import \
     TEST_VECTOR \
     VERSION \
...
```

**Review by cjpatton — APPROVED** [review]
Let's do it!

File: poc/prg.sage (hunk 1 above)
Comment (divergentdave) [reply poc/prg.sage:11]: I think I'll just run `isort` on this. It'll use parentheses instead of backslash escaping newlines, FWIW.

Comment (divergentdave) [reply poc/prg.sage:11]: I think I'm going to leave the import statements as-is for now. In the future, maybe we could add `autopep8` and `isort` to a Makefile target, and automate this formatting.

Comment (cjpatton) [reply poc/prg.sage:11]: Sounds like  plan to me.

**Pull request merged**